app/code/Magento/AsynchronousOperations/Model/SaveMultipleOperations.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AsynchronousOperations\Model;
+
+use Magento\AsynchronousOperations\Api\Data\OperationInterface;
+use Magento\AsynchronousOperations\Api\SaveMultipleOperationsInterface;
+use Magento\AsynchronousOperations\Model\ResourceModel\Operation as OperationResource;
+use Magento\Framework\Exception\CouldNotSaveException;
+
+/**
+ * Implementation for saving multiple operations
+ */
+class SaveMultipleOperations implements SaveMultipleOperationsInterface
+{
+
+    /**
+     * @var OperationResource
+     */
+    private $operationResource;
+
+    /**
+     * BulkSummary constructor.
+     *
+     * @param OperationResource $operationResource
+     */
+    public function __construct(
+        OperationResource $operationResource
+    ) {
+        $this->operationResource = $operationResource;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function execute(array $operations): void
```

Comment:
You have declared **OperationInterface[]** expected type for '$operations', but in current implementation it will works correctly only with data in arrays.

Please keep contract on **OperationInterface** or do not make current service as API.